testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinatorTests.cs

```diff
@@ -1,23 +1,28 @@
 using System.Collections;
+using System.IO;
 using System.Linq;
 using NUnit.Framework;
 using UnityEngine;
 using UnityEngine.TestTools;
 
 namespace Unity.Netcode.MultiprocessRuntimeTests
 {
-    [TestFixture(1)]
-    [TestFixture(2)]
+    [TestFixture(1, new string[] { "default-win:test-win" })]
+    [TestFixture(3, new string[] { "default-win:test-win", "default-mac:test-mac-2", "default-win:test-win-2" })]
```

Comment:
mmm as a test writer, I don't see myself writing this :/
1 - we don't need workerCount if we have the list's count
2 - those strings should be const somewhere
3 - could it be possible to generate those combinations and as a dev I'd just have to worry about whether the test will be 1 vs 1+ worker (as a test writer, writing a test for 2 workers and 3 or 4 workers doesn't change much, it's the same for loop :P )?
4 - I'm guessing we'll have more than windows and mac eventually? (mobile? console?) --> combinations will become crazy!

Reply:
those issues don't all need to be addressed in this PR, but they'll be important to address to make test writing easier.

Reply:
Okay, that all makes sense. I don't have a great solution for how to specify the platforms combinations to run on so I'll come back to this.

Reply:
I don't understand how test fixture annotation/decoration works. If I just remove the numbers and leave the string array I get errors that I can't understand. I'll share them in slack and we can figure it out there, I hope.